docs/source/release-notes.rst

```diff
@@ -1,6 +1,10 @@
 Release Notes
 =============
 
+Version 0.4.1
+_____________
+- FIX: `date` and `datetime` types are handled according OpenAPI specs in query string parameters.
+
```

Comment:
Add "This is not a breaking change but consider to check the that for existing date/datetime queryargs, value format is going to be validated at request from this version."

Reply:
Recheck the message with gramarly.